src/components/Calculator.js

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import calculate from './logic/calculate';
+
+function Calculator() {
+  const [dataNow, newData] = useState({});
+
+  function getData(event) {
+    const button = event.target.name;
+    newData(calculate(dataNow, button));
+  }
+
+  function Button({ values, className }) {
+    return (
+      <button type="button" name={values} className={className} onClick={getData}>
+        {values}
+      </button>
+    );
+  }
+
+  Button.propTypes = {
+    values: PropTypes.oneOfType([PropTypes.string, PropTypes.number]).isRequired,
+    className: PropTypes.oneOfType([PropTypes.string, PropTypes.number]).isRequired,
+  };
```

Comment:
- [ ] As I mentioned in my above comment, please put your `components` into their respective file and folders, to make your codebase more neat and clean, easy to maintain and definitely help to reduce the number of lines of code in each file.